bundles/ui/org.eclipse.smarthome.ui.test/src/test/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImplTest.java

```diff
@@ -162,7 +162,7 @@ public void getLabel_labelWithDate() throws ItemNotFoundException {
         when(w.getLabel()).thenReturn(testLabel);
         when(w.getItem()).thenReturn("Item");
         when(registry.getItem("Item")).thenReturn(item);
-        when(item.getState()).thenReturn(new DateTimeType("2011-06-01T00:00:00"));
+        when(item.getState()).thenReturn(new DateTimeType("2011-06-01T00:00:00Z"));

```

Comment:
Does it still accept the old format like `2011-06-01T00:00:00`, i.e. is it backward-compatible?